src/index.js

```diff
@@ -22,25 +22,25 @@ function addToLocalStorage(scores) {
   localStorage.setItem('scores', JSON.stringify(scores));
 }
 
-function appendScoresToList() {
+function appendScoresToList(scores) {
```

Comment:
You have a few old-school functions on this js file. Kindly make sure to use arrow functions instead to meet the project requirements 🙌

Reply:
Thanks for the feedback @mateo951, I'll definitely change that. 👌🏻